test/core/channel/channel_stack_builder_test.cc

```diff
@@ -57,70 +53,57 @@ void CallDestroyFunc(grpc_call_element* /*elem*/,
                      const grpc_call_final_info* /*final_info*/,
                      grpc_closure* /*ignored*/) {}
 
-bool g_replacement_fn_called = false;
-bool g_original_fn_called = false;
-
-void SetReplacementFnCalled(grpc_channel_stack*, grpc_channel_element*) {
-  g_replacement_fn_called = true;
-}
-
-void SetOriginalFnCalled(grpc_channel_stack*, grpc_channel_element*) {
-  g_original_fn_called = true;
+const grpc_channel_filter* FilterNamed(const char* name) {
```

Comment:
Suggest calling this something like `CreateFilter()`.

Reply:
eh... it'd be GetOrCreateFilterNamed()... and I think that overall hurts the readability of the tests?